test/integration/sourcemap/snapshot.spec.ts

```diff
@@ -79,7 +79,7 @@ describe("Source map snapshot tests", () => {
                     const result = parts.join("\n") + "\n";
 
                     // Uncomment next line to update snapshots
-                    // fse.writeFileSync(sample, result, { encoding: "utf-8" });
+                    fse.writeFileSync(sample, result, { encoding: "utf-8" });
```

Comment:
This should be commented right?

Reply:
Right, will fix.

Reply:
Addressed in https://github.com/ConsenSys/solc-typed-ast/pull/118/commits/71fe51fc8f15cb2fb7dedb1e7f639465088f3425